articles/ros_node_idl.md

```diff
@@ -70,8 +70,20 @@ If ROS 2 provided a way for upstream package authors to specify the interface re
 ### Other motivations
 
 Outside of security, there are several fascinating possibilities unlocked by having such an interface.
+
 For example, consider how this could impact [ROS 2 launch][launch_ros].
-It would be able to statically (i.e. before running anything) determine if parameter names or remappings are incorrect, among other similar sanity checks.
+Benefiting from the declared interface(s), it would be able to execute many kind of static asserts (i.e. at launch-time, before running anything) upon the whole system to be launched.
```

Comment:
```suggestion
Benefiting from the declared interface(s), it would be able to execute many kind of static assertions (i.e. at launch-time, before running anything) upon the whole system to be launched.
```